cps-v2/enrollments.go

```diff
@@ -0,0 +1,123 @@
+package cps
+
+import (
+  "github.com/akamai/AkamaiOPEN-edgegrid-golang/edgegrid"
+  "github.com/akamai/AkamaiOPEN-edgegrid-golang/client-v1"
+)
+
+// Enrollments represents a CPS enrollments list
+type Enrollments struct {
+  Enrollments []Enrollment `json:"enrollments"`
+}
+
+// NewEnrollments creates an empty Enrollments
+func NewEnrollments() *Enrollments {
+  e := &Enrollments{}
+  return e
+}
+
+// Get populates Enrollments with the list of avaiable enrollments
+//
+// API Docs: https://developer.akamai.com/api/luna/cps/resources.html#getenrollments
+func (e *Enrollments) Get(config edgegrid.Config) error {
```

Comment:
For consistency: 

- `config` should be a package-level variable, `cps.Config` set either manually, or via an `Init()` function (see: [here](https://github.com/akamai/AkamaiOPEN-edgegrid-golang/blob/master/configdns-v1/service.go))
- `Get()` should called `GetEnrollments()`

Reply:
I have an idea why you want the config this way however can you provide some details on why the config should be package-level and why it should be manually set? Are you open to exploring different ways of handling the config?